### PR TITLE
[Merged by Bors] - Ipv6 bootnodes update

### PR DESCRIPTION
**Pull request by AgeManning**
We now officially have ipv6 support. The mainnet bootnodes have been updated to support ipv6. This PR updates lighthouse's internal bootnodes for mainnet to avoid fetching them on initial load.

### CODE REVIEW

**Comment by divagant-martian** [comment]
can connect to the two new bootnodes just fine :zap: 

but you forgot to update the tests :stuck_out_tongue:

**Comment by AgeManning** [comment]
Yeah, i changed that test. Not sure what its testing, but I think updating ENRs we shouldn't have to update this random test with the exact value. 

Changed the test, to just make sure the count is the same.

File: common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml
Comment (one-three-three-seven) [inline common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml]: There is an unnecessary space after "enr:-"

Comment (AgeManning) [reply common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml]: Yep, thanks for the pickup

**Review by paulhauner — APPROVED** [review]
Looks good to me, happy to merge after we have @antondlr's blessing.

**Comment by divagant-martian** [comment]
this makes me happy :smiling_face_with_tear:

**Pull request closed without merging**
